spec/data-model/README.md

```diff
@@ -110,18 +110,16 @@ interface CatchallKey {
 ## Patterns
 
 Each `Pattern` represents a linear sequence, without selectors.
-Each element of the `body` array MUST either be a non-empty string or an `Expression` object.
+Each element of the `Pattern` MUST either be a non-empty string or an `Expression` object.
 String values represent literal _text_,
 while `Expression` wraps each of the potential _expression_ shapes.
-The `body` strings are the "cooked" _text_ values, i.e. escape sequences are processed.
+The `Pattern` strings are the "cooked" _text_ values, i.e. escape sequences are processed.
```

Comment:
The whole "cooking" thing seems just a bit informal to me. Here we refer to "`Pattern` strings" and I think that points to the problem. Perhaps:

```suggestion
String values represent literal _text_.
String values include all processing of the underlying _text_ values, that is, after escape sequences have been processed.

`Expression` values wrap each of the potential _expression_ shapes.

```

Reply:
Added this with a few tweaks.